models/schedule/kinds/interval.py

```diff
@@ -0,0 +1,20 @@
+from datetime import datetime
+
+
+class Interval:
+    def __init__(self, time, start, end):
+        self.time = time
+        self.start = start
+        self.end = end
+        self.next_execution = datetime.min
```

Comment:
```suggestion
    def __init__(self, config):
        self.next_execution = datetime.min
        self.time = timedelta(minutes=config["time"])
        self.start = parse_time(config["start"])
        self.end = parse_time(config["end"])

    def parse_time(self, time):
        return datetime.strptime(time, "%H:%M").time()
    end
```

Seguindo a ideia de passar o schedule e aqui extrair as informações que precisa

Reply:
Considerado no ajuste do factory